integrated_channels/moodle/migrations/0032_auto_20231208_2345.py

```diff
@@ -0,0 +1,45 @@
+# Generated by Django 3.2.23 on 2023-12-08 23:45
```

Comment:
We can skip this migration for now. Let's first check whether the transmissions work for rest of the customers and then we can add the migration later on.

Reply:
@sameenfatima78 updated.